zookeeper-server/src/main/java/org/apache/zookeeper/version/util/VerGen.java

```diff
@@ -198,7 +198,7 @@ public static void main(String[] args) {
             } else {
                 rev = rev.trim();
             }
-            generateFile(new File("."), version, rev, args[2]);
+            generateFile(new File("../../../target/generated-sources/java"), version, rev, args[2]);
```

Comment:
It is better to have this path passed as a parameter, this way it will be easier to drive this code from the pom.xml